readme.md

```diff
@@ -54,6 +64,7 @@ console.log(results);
 
 Your Answer:
 ```text
+Since "find" is synonymous with "where" in SQL, the query returns an array, thus if there were multiple documents with the name "Bob", all of these objects would be returned.
```

Comment:
That's true, but the bigger issue is that `.find` is asynchronous, so the `console.log` would happen before the `.find` has completed.